### ISSUE TITLE
[Bug]: CommandBar Required ARIA children role not present if the CommandBar is empty

### ISSUE THREAD

**Issue by EbramTawfik** [microsoft/fluentui#23102]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.22000
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz
    Memory: 26.37 GB / 47.71 GB
  Browsers:
    Edge: Spartan (44.22000.120.0), Chromium (101.0.1210.47)
    Internet Explorer: 11.0.22000.120
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codepen.io/EbramTawfik/pen/JjpJozZ?editors=1010

## Actual Behavior
CommandBar doesn't pass the Accessibility Insights for Web test if it's empty with this error:

[WCAG 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships) 

Fix the following:
- Required ARIA children role not present: group, menuitemradio, menuitem, menuitemcheckbox

![image](https://user-images.githubusercontent.com/12788360/169432370-8c9db8bf-59c1-4113-a8a0-f6f46a86965f.png)

**CodePen Example:**
https://codepen.io/EbramTawfik/pen/JjpJozZ?editors=1010
![image](https://user-images.githubusercontent.com/12788360/169432584-753b4ee8-1b95-4610-910d-ebd9197e95a1.png)

Steps to reproduce:
1- install Accessibility Insights for web https://accessibilityinsights.io/downloads/
2- open the CodePen Example: https://codepen.io/EbramTawfik/pen/JjpJozZ?editors=1010
3- run FastPass from the Accessibility Insights for Web extension:
![image](https://user-images.githubusercontent.com/12788360/169433041-f8690046-7c74-4abd-ae61-4931d8f6181b.png)

## Expected Behavior
Adding the missing fields (role and aria-label) to the div child when the CommandBar is empty.

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by gouttierre** [comment]
Thanks for filing this issue with us. I have added this to our accessibility queue for further investigations. To set expectations, the developers will review this issue once capacity allows.
